sql/src/main/java/io/crate/analyze/OrderBy.java

```diff
@@ -195,8 +195,13 @@ public int hashCode() {
     @Override
     public String toString() {
         StringBuilder sb = new StringBuilder("OrderBy{");
-        for (int i = 0; i < orderBySymbols.size(); i++) {
-            Symbol symbol = orderBySymbols.get(i);
+        prettyPrint(sb, orderBySymbols, reverseFlags, nullsFirst);
+        return sb.toString();
+    }
+
+    public static StringBuilder prettyPrint(StringBuilder sb, List<Symbol> symbols, boolean[] reverseFlags, Boolean[] nullsFirst) {
```

Comment:
dunno if i would put a method on the objects. also i  think hat using it for toString might lead to breaking toStrings in the debugger. i would rather keep it in a visitor